news/2024/2024-09-19-kyoku-original-song-and-beatmapping-contest-results.md

```diff
@@ -0,0 +1,98 @@
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://assets.ppy.sh/media/news/kyoku-2024-3rd-place.mp4" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Congratulations to Team RGB as they scored themselves 3rd place with a final score of **84.50**!
+
+### 2nd place - Zero Blunders
+
+Team Zero Blunders has put their best foot forward in this contest with a genre mash-up song featuring many osu! references. [*who's afraid?*](https://osu.ppy.sh/beatmapsets/2242420#osu/4766280) by fur:trash (also known as [Nihkee](https://osu.ppy.sh/beatmapsets/2242420#osu/4766280)) offers many chances for [Mysty](https://osu.ppy.sh/users/10210657), [Le Mirai](https://osu.ppy.sh/users/13646997), [hakashii](https://osu.ppy.sh/users/11688893) & [[-Evil-]](https://osu.ppy.sh/users/10234313) to express their mapping ideas through music genres such as dariacore, dubstep & hyperpop, creating a balanced map that is both fun to look at and play.
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://assets.ppy.sh/media/news/kyoku-2024-2nd-place.mp4" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Congratulations to Zero Blunders for managing to lock in a total score of **86.00**, securing 2nd place and 1 contest point for all team members.
+
+### 1st place - cosmo/LOGIΛ
+
+Team cosmo/LOGIΛ have managed to concoct what can best be described as a Grand Finals Tiebreaker map on steroids. [Stellar Evolution](https://osu.ppy.sh/beatmapsets/2242303) from [airlemoneX](https://soundcloud.com/airlemonex) has hints of [Camellia](https://osu.ppy.sh/beatmaps/artists/31) as the inspiration behind the track, with expert mappers [Lulu-](https://osu.ppy.sh/users/4201715), [Slifer](https://osu.ppy.sh/users/15084122) and [Halgoh](https://osu.ppy.sh/users/4109923) fully playing into it with a focus on visuals that do not compromise gameplay.
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://assets.ppy.sh/media/news/kyoku-2024-1st-place.mp4" frameborder="0" allowfullscreen></iframe>
+</div>
+
+With a final score of **88.25**, cosmo/LOGIΛ have managed to achieve first place, earning badges along with 2 contest points. Congratulations!
+
+![](/wiki/Contests/Kyoku/2024/img/badge.png)
+
+## Submissions
+
+It is highly recommended that you check out all the entries as every single team put blood, sweat and tears into their submissions. You can either check out the full VOD from the results reveal below, or check out and download individual submissions and give them a rating yourself.
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/kzsxu8254u4" frameborder="0" allowfullscreen></iframe>
```

Comment:
the beginning includes info directed towards teams which isn't very relevant here ([mentioned in the thread](https://discord.com/channels/188630481301012481/1237198052054274058/1279934527074209913))

```suggestion
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/kzsxu8254u4?start=56" frameborder="0" allowfullscreen></iframe>
```